modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -432,4 +436,260 @@ public static ReadOnlyBooleanProperty accessibilityActiveProperty() {
+
+        /**
+         * Returns the {@code Double} instance to which the specified key is mapped.
+         *
+         * @param key the key
+         * @throws NullPointerException if {@code key} is null
+         * @throws IllegalArgumentException if the key is not mapped to a {@code Double} instance
+         * @return the {@code Double} instance to which the key is mapped, or {@code Optional.empty()}
+         *         if no mapping exists for the specified key
+         */
+        Optional<Double> getDouble(String key);
+
+        /**
+         * Returns the {@code Boolean} instance to which the specified key is mapped.
+         *
+         * @param key the key
+         * @throws NullPointerException if {@code key} is null
+         * @throws IllegalArgumentException if the key is not mapped to a {@code Boolean} instance
+         * @return the {@code Boolean} instance to which the key is mapped, or {@code Optional.empty()}
+         *         if no mapping exists for the specified key
+         */
+        Optional<Boolean> getBoolean(String key);
+
+        /**
+         * Returns the {@code String} instance to which the specified key is mapped.
+         *
+         * @param key the key
+         * @throws NullPointerException if {@code key} is null
+         * @throws IllegalArgumentException if the key is not mapped to a {@code String} instance
+         * @return the {@code String} instance to which the key is mapped, or {@code Optional.empty()}
+         *         if no mapping exists for the specified key
+         */
+        Optional<String> getString(String key);
+
+        /**
+         * Returns the {@code Color} instance to which the specified key is mapped.
+         *
+         * @param key the key
+         * @throws NullPointerException if {@code key} is null
+         * @throws IllegalArgumentException if the key is not mapped to a {@code Color} instance
+         * @return the {@code Color} instance to which the key is mapped, or {@code Optional.empty()}
+         *         if no mapping exists for the specified key
+         */
+        Optional<Color> getColor(String key);
+
+        /**
+         * Returns the {@code Paint} instance to which the specified key is mapped.
+         *
+         * @param key the key
+         * @throws NullPointerException if {@code key} is null
+         * @throws IllegalArgumentException if the key is not mapped to a {@code Paint} instance
+         * @return the {@code Paint} instance to which the key is mapped, or {@code Optional.empty()}
+         *         if no mapping exists for the specified key
+         */
+        Optional<Paint> getPaint(String key);
+
+        /**
+         * Returns the value to which the specified key is mapped.
+         *
+         * @param <T> the type of the value
+         * @param key the key
+         * @param type the type of the value
+         * @throws NullPointerException if {@code key} is null
```

Comment:
```suggestion
         * @throws NullPointerException if {@code key}, or {@code type} is null
```
Or my new favourite for this kind of thing:
```suggestion
         * @throws NullPointerException when any argument is null
```